draft-ietf-quic-http.md

```diff
@@ -86,25 +86,28 @@ code and issues list for this draft can be found at
 
 # Introduction
 
-HTTP semantics are used for a broad range of services on the Internet. These
-semantics have commonly been used with two different TCP mappings, HTTP/1.1 and
-HTTP/2.  HTTP/3 supports the same semantics over a new transport protocol, QUIC.
+HTTP semantics {{!SEMANTICS=I-D.ietf-httpbis-semantics}} are used for a broad
+range of services on the Internet. These semantics have commonly been used with
+two different TCP mappings, HTTP/1.1 and HTTP/2.  HTTP/3 supports the same
+semantics over a new transport protocol, QUIC.
 
 ## Prior versions of HTTP
 
-HTTP/1.1 is a TCP mapping which uses whitespace-delimited text fields to convey
-HTTP messages.  While these exchanges are human-readable, using whitespace for
-message formatting leads to parsing difficulties and workarounds to be tolerant
-of variant behavior. Because each connection can transfer only a single HTTP
-request or response at a time in each direction, multiple parallel TCP
-connections are often used, reducing the ability of the congestion controller to
-accurately manage traffic between endpoints.
-
-HTTP/2 introduced a binary framing and multiplexing layer to improve latency
-without modifying the transport layer.  However, because the parallel nature of
-HTTP/2's multiplexing is not visible to TCP's loss recovery mechanisms, a lost
-or reordered packet causes all active transactions to experience a stall
-regardless of whether that transaction was impacted by the lost packet.
+HTTP/1.1 {{?HTTP11=I-D.ietf-httpbis-messaging}} is a TCP mapping which uses
+whitespace-delimited text fields to convey HTTP messages.  While these exchanges
+are human-readable, using whitespace for message formatting leads to parsing
+difficulties and workarounds to be tolerant of variant behavior. Because each
```

Comment:
```suggestion
complexity and motivates tolerance of variant behavior. Because each
```

Motivates may not be the best word, but requires was my other suggestion, and that's not quite accurate.